apps/web/src/pages/home/index.tsx

```diff
@@ -124,21 +122,22 @@ function convertWindSpeed(
 }
 
 const InternalHome = observer(() => {
+  const locale = useCurrentLocale();
   const [isMoreInfoCollapsibleOpen, setIsMoreInfoCollapsibleOpen] =
     useState(false);
   const [isMoreWarningsCollapsibleOpen, setIsMoreWarningsCollapsibleOpen] =
     useState(false);
-  const { locale, query, replace, isReady } = useRouter();
+  const { query, replace, isReady } = useRouter();
 
-  const cityById = useQuery(
-    convexApi.getCity.findCityById,
-    typeof query.cityId === "string" ? { id: parseInt(query.cityId) } : "skip",
+  const { data: findCityByIdData = [] } = api.search.findCityById.useQuery(
+    typeof query.cityId === "string"
+      ? { id: parseInt(query.cityId) }
+      : skipToken,
   );
 
   useEffect(() => {
-    const cityByIdIsLoading =
-      cityById === undefined && typeof query.cityId === "string";
-    if (!cityByIdIsLoading && !cityById && isReady) {
+    if (Array.isArray(findCityByIdData) && isReady) {
+      console.log("moin meister");
```

Comment:
Please remove the console log here, i think you dont need this console log.

Reply:
Nice, thank you. Was just for testing purposes